autogtp/Job.cpp

```diff
@@ -17,13 +17,12 @@
 */
 
 #include "Job.h"
-#include "Game.h"
-#include "Management.h"
-#include <QTextStream>
-#include <chrono>
-
 #include <QFile>
+#include <QTextStream>
 #include <QThread>
+#include "Game.h"
```

Comment:
the includes of your files should go before the standard includes.

Reply:
That would be inconsistent with how it's done in `src`.

Reply:
That's because it's the complementary header file.

Reply:
`Game.h` gets included first because it's the header file belonging to `Game.cpp`.